sphinx_docs/source/alcf.rst

```diff
@@ -52,6 +51,10 @@ https://www.alcf.anl.gov/support/user-guides/polaris/queueing-and-running-jobs/j
 
 For production jobs, you submit to the ``prod`` queue.
 
+For debugging jobs, there are two options: the ``debug`` queue for only one node use jobs, and 
```

Comment:
`debug` allows 1 to 2 nodes per job, with a max of 24 nodes in use across the entire system; `debug-scaling` allows 1 to 10 nodes per job, with a max of 1 job in any state per user (see https://docs.alcf.anl.gov/polaris/running-jobs/#queues)

Reply:
That seems to be recent, I will correct this but I will not mention the max of 24 nodes; I will refer to the webpage instead.